articles/forward-and-backward-chaining-in-ai/index.md

```diff
@@ -0,0 +1,119 @@
+## Forward and Backward Chaining in Artificial Intelligence
+
+Backward and forward chaining are methods of reasoning that exist in the Expert System Domain of artificial intelligence. 
+
+### Introduction to the Expert System
+Backward and forward chaining originate from the expert system. The expert system has three main components: user interface, inference engine, and knowledge base. 

```

Comment:
What is an expert system? You mention Chaining techniques stem from expert systems. Explain that first.

Reply:
Make it clear that backward and forward chaining techniques are used in expert systems. Give examples of where this would be used.

Reply:
Mentioning applications ensure the reader's curiosity is retained.

Reply:
I have revised accordingly. I have explained what an expert system is. I have also provided examples in this section.